vllm/distributed/parallel_state.py

```diff
@@ -45,6 +49,8 @@ def init_distributed_environment(
     local_rank: int = -1,
     backend: str = "nccl",
 ):
+    logger.info(f"{world_size=} {rank=} {local_rank=} "
```

Comment:
```suggestion
    logger.debug(f"{world_size=} {rank=} {local_rank=} "
```

Reply:
i see the default value is debug: 

https://github.com/vllm-project/vllm/blob/5c2e66e4871917c5d59cc4a8b89ef53e690e9bd9/vllm/logger.py#L58